src/SFA.DAS.AssessorService.Infrastructure/SFA.DAS.AssessorService.Infrastructure.csproj

```diff
@@ -17,8 +17,8 @@
   </ItemGroup>
 
   <ItemGroup>
+	<FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Tab

Reply:
These appear consistent on my local copy. All lines use spaces rather than tabs.

Reply:
Yeah but they don't line up so something is wrong